src/v2/ion/i18nTransformer.js

```diff
@@ -93,9 +93,11 @@ const I18N_OVERRIDE_MAPPINGS = {
   'authenticator-verification-data.okta_verify.authenticator.methodType.totp': 'oie.okta_verify.totp.title',
 
   'authenticator-enrollment-data.phone_number.authenticator.phoneNumber': 'mfa.phoneNumber.placeholder',
+  'authenticator-enrollment-data.phone_number.authenticator.methodType.sms': 'oie.phone.enroll.sms.label',
+  'authenticator-enrollment-data.phone_number.authenticator.methodType.voice': 'oie.phone.enroll.voice.label',
 
   'enroll-authenticator.okta_password.credentials.passcode': 'oie.password.passwordLabel',
-  'enroll-authenticator.phone.credentials.passcode': 'mfa.challenge.enterCode.placeholder',
+  'enroll-authenticator.phone_number.credentials.passcode': 'mfa.challenge.enterCode.placeholder',
```

Comment:
@pradeepdewda-okta - I was wondering if you can take a look at this change. The previous key `enroll-authenticator.phone.credentials.passcode` didn't make sense as there's no `phone` named authenticator (it seems to be `phone_number` instead). I have also added a test for this.

Reply:
@anipendakur-okta this looks good to me..